pygfx/helpers/_axes.py

```diff
@@ -79,7 +79,12 @@ def __init__(self, size=1.0, thickness=2):
             arrow_head.local.position = pos
             # offset by half of height since the cones
             # are centered around the origin
-            arrow_head.local.position += arrow_size / 2 * la.vec_normalize(pos)
+            # Do not use +=
+            # see: https://github.com/pygfx/pygfx/issues/651
+            arrow_head.local.position = (
+                arrow_head.local.position +
+                arrow_size / 2 * la.vec_normalize(pos)
```

Comment:
Well, that's an annoying and surprising side effect of this change! :/ I guess I knew that ndarrays overload this to be in-place...